test/datatype/reduce_local.c

```diff
@@ -50,19 +59,9 @@ static int do_ops[12] = {
 static int verbose = 0;
 static int total_errors = 0;
 
-#define max(a, b)               \
-    ({                          \
-        __typeof__(a) _a = (a); \
-        __typeof__(b) _b = (b); \
-        _a > _b ? _a : _b;      \
-    })
-
-#define min(a, b)               \
-    ({                          \
-        __typeof__(a) _a = (a); \
-        __typeof__(b) _b = (b); \
-        _a < _b ? _a : _b;      \
-    })
+#define max(a, b) (a) > (b) ? (a) : (b)
```

Comment:
`max()` isn't used, and `min()` looks like it isn't used in a way where this implementation breaks, so I'm ok with it.

Reply:
`typeof` is not C11 and the generic types aren't needed here.